app/src/main/java/no/usn/mob3000/ui/screens/chess/train/opening/CreateOpeningScreen.kt

```diff
@@ -59,6 +62,7 @@ fun CreateOpeningScreen(
                 onValueChange = { description = it },
                 label = { Text(stringResource(R.string.opening_create_prompt_description)) },
                 modifier = Modifier.fillMaxWidth(),
+                textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground,),
```

Comment:
Unnecessary internal trailing comma. There are no additional parameters inputted, and there shouldn't be trailing commas like this.